internal/ui/esbuild.go

```diff
@@ -57,14 +58,14 @@ var buildOptions = api.BuildOptions{
 	},
 	Plugins: []api.Plugin{
 		{
-			Name: "Reset Plugin",
+			Name: "Reset dist & Generate Tailwind CSS",
 			Setup: func(setup api.PluginBuild) {
 				setup.OnStart(func() (api.OnStartResult, error) {
```

Comment:
The `buildOptions` variable definition is too long (~100 lines).

Consider creating helper setup functions and a separate variable for the entry points.